crates/ruff_python_ast/src/snapshots/ruff_python_ast__visitor__tests__compare.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: crates/ruff_python_ast/src/visitor.rs
```

Comment:
Most of these snapshots are new coverage of previous behavior with tests duplicated from the pre-order visitor.